### PR TITLE
Added the DebugUtil with some methods for debugging

### PR DESCRIPTION
**Pull request by pveentjer**
I added this DebugUtil because often I need to see what happened (e.g. whole called a certain method) in runtime, but can't rely on a debugger.

### CODE REVIEW

File: logs/provisioner.log [hunk 1]
```diff
@@ -0,0 +1,11 @@
+INFO  2015-02-11 13:12:05,583 [main] com.hazelcast.stabilizer.provisioner.Provisioner: Hazelcast Stabilizer Provisioner
```
Comment (eminn) [inline logs/provisioner.log]: I think this file is a leftover. Isn't it ?

**Comment by eminn** [comment]
Also there is an empty agents.txt file which is also is not related I think.

File: logs/provisioner.log (hunk 1 above)
Comment (pveentjer) [inline logs/provisioner.log]: Jup. Thanks for spitting it.

**Comment by pveentjer** [comment]
verify

File: hazelcast/src/main/java/com/hazelcast/util/DebugUtil.java [hunk 2]
```diff
@@ -0,0 +1,86 @@
+package com.hazelcast.util;
```
Comment (bilalyasar) [inline hazelcast/src/main/java/com/hazelcast/util/DebugUtil.java]: missing headers

**Comment by pveentjer** [comment]
verify

**Comment by pveentjer** [comment]
Since it only adds code for debugging and doesn't change any existing classes, I'm going to merge this one.

**Pull request merged**